src/flowcean/strategies/deployment.py

```diff
@@ -0,0 +1,50 @@
+import polars as pl
+from flowcean.core.environment.active import ActiveEnvironment
+from flowcean.core.environment.incremental import IncrementalEnvironment
+from flowcean.core.transform import ChainedTransforms
+
+from flowcean.core.model import ModelWithTransform
+
+def deploy(
+    environment: ActiveEnvironment | IncrementalEnvironment,
+    config: list[str],
+    model: ModelWithTransform,
+    input_transforms: ChainedTransforms,
```

Comment:
This type bound probably only needs to be `Transform`
```suggestion
    input_transforms: Transform,
```

Reply:
Done